doc/classes/Environment.xml

```diff
@@ -1,8 +1,15 @@
 <?xml version="1.0" encoding="UTF-8" ?>
-<class name="Environment" inherits="Resource" category="Core" version="3.0.alpha.custom_build">
+<class name="Environment" inherits="Resource" category="Core" version="3.0">
 	<brief_description>
+		Resource for environment nodes (like [WorldEnvironment]) that define multiple rendering options.
 	</brief_description>
 	<description>
+		Resource for environment nodes (like [WorldEnvironment]) that define multiple environment operations (such as background [Sky] or [Color], ambient light, fog, depth-of-field...).	These parameters affect the final render of the scene. The order of these operations is:
```

Comment:
It defines:
-Sky mode
-HDR and Tonemapping
-Post Processing effects
-Color correction adjustments